app/models/invenio_rdm_record_converter.rb

```diff
@@ -112,7 +113,13 @@ def file_permissions
     permission_data = Hash.new { |h,k| h[k] = [] }
```

Comment:
I get that this is an older bit of code, but we should really change this to `permission_data = Hash.new` or `permission_data = {}`. Kind of verbose and distracting the way it is.